app/styles/templates/organisations/_new.scss

```diff
@@ -0,0 +1,149 @@
+$dropdown-body: #000000;
+$input-border: #8091a9;
+$white: #fff;
+$background-color: #002352;
+$error-background: #f46769;
+
+.header-container {
```

Comment:
Can you please scope all the classes to to this page. We can't keep putting classes at the top level, it creates too many conflicts across the project

Reply:
this applies to all CSS files btw